src/peer_loop.rs

```diff
@@ -1882,6 +1889,19 @@ impl PeerLoopHandler {
         // feature to have for testing purposes.
         res
     }
+
+    /// Register graceful peer disconnection in the global state.
```

Comment:
Comment here could be a bit more descriptive maybe. I like how the field on the networking state is very explicit.